test_scripts/Polices/Policy_Table_Update/027_ATF_P_TC_HMILvl_on_PTU_affected_in_FULL_or_LIMITED.lua

```diff
@@ -182,19 +180,15 @@ function Test:RegisterSecondApp()
       self.mobileSession2:ExpectResponse(correlationId2, { success = true })
       self.mobileSession2:ExpectNotification("OnPermissionsChange")
     end)
-end
-
-function Test:ActivateFirstApp()
-  commonSteps:ActivateAppInSpecificLevel(self, HMIAppID)
+  self.mobileSession2:ExpectNotification("OnHMIStatus",{ systemContext = "MAIN", hmiLevel = "NONE"})
```

Comment:
@istoimenova, According to req-t in description applications have to be in FULL/LIMITED HMI levels
Currently they will be in FULL/NONE